orchagent/vxlanorch.cpp

```diff
@@ -360,6 +428,9 @@ bool VxlanTunnel::createTunnel(MAP_T encap, MAP_T decap)
         ids_.tunnel_term_id = create_tunnel_termination(ids_.tunnel_id, ips, ip, gVirtualRouterId);
         active_ = true;
         tunnel_map_ = { encap, decap };
+
+        ids_.tunnel_bridge_port_id = create_tunnel_bridge_port(ids_.tunnel_id);
```

Comment:
VNetBitmapObject::getBridgeInfoByVni also creates a bridge_port for the tunnel and also calls createTunnel. Isnt It is better to remove the bridge_port creation and consolidate bridge port creations in createTunnel to avoid duplicate bridge port creations ?

Reply:
class VNetBitmapObject is not defined when we writing the codes. Yes, it is better to consolidate bridge port creations in createTunnel. We wiil remove the bridge_port creation in VNetBitmapObject::getBridgeInfoByVni, only record the bridge port id to info.bridge_port_tunnel_id. Thanks.